src/assets/scss/app.scss

```diff
@@ -0,0 +1,19 @@
+@import "~bulma";
+@import "~bulma-timeline";
+
+.navbar {
+  border-bottom: 1px solid #eee;
+  margin-bottom: 8px;
+}
+
+.gh-username img{
+  width: 32px;
+}
+
+#profile-bio{
```

Comment:
- [ ] please don't use the ID for a selector
- [ ] space before the curly brace